src/Entities/RulesEngine/Actions/RequestKyc.php

```diff
@@ -0,0 +1,291 @@
+<?php
+/**
+ * This source file is proprietary and part of Rebilly.
+ *
+ * (c) Rebilly SRL
+ *     Rebilly Ltd.
+ *     Rebilly Inc.
+ *
+ * @see https://www.rebilly.com
+ */
+
+namespace Rebilly\Entities\RulesEngine\Actions;
+
+use DomainException;
+use Rebilly\Entities\RulesEngine\RuleAction;
+
+/**
+ * Class RequestKyc.
+ */
+final class RequestKyc extends RuleAction
+{
+    public const UNEXPECTED_POLICY = 'Unexpected policy. Only %s policies are supported';
+
+    public const EXCLUDE_POLICY_NONE = 'none';
+
+    public const EXCLUDE_POLICY_CUSTOMERS_WITH_ACCEPTED_DOCUMENT = 'customers-with-accepted-document';
+
+    public const EXCLUDE_POLICY_CUSTOMERS_WITH_DOCUMENT = 'customers-with-document';
+
+    public const PROMPT_POLICY_BEFORE_TRANSACTION_PROCESS = 'before-transaction-process';
+
+    public const PROMPT_POLICY_AFTER_TRANSACTION_PROCESS = 'after-transaction-process';
+
+    public const REJECTED_BEFORE_TRANSACTION_PROCESS_POLICY_DECLINED = 'process-transaction';
+
+    public const REJECTED_BEFORE_TRANSACTION_PROCESS_POLICY_PROCESS_TRANSACTION = 'process-transaction';
```

Comment:
this constant has the same value as REJECTED_BEFORE_TRANSACTION_PROCESS_POLICY_DECLINED and located with it in the same array, it will be impossible to determine what exactly constant is used